frontend/src/Admin/components/NavBar.js

```diff
@@ -49,6 +49,32 @@ export default class NavBar extends Component {
           <NavbarToggler onClick={this.toggle} />
           <Collapse isOpen={this.state.isOpen} navbar>
             <Nav className='ml-auto' navbar>
+              <NavItem>
+                <NavLink tag={Link} to='/'>
+                  Dashboard
```

Comment:
I would say change this to shows dashboard or something

Reply:
Changed to "Shows Dashboard"